draft-twi-reference-architecture.md

```diff
@@ -217,7 +217,10 @@ Executing the Workload inside a Trusted Execution Environment can lower the risk
 
 # Security Considerations
 
-<cref>TODO</cref>
+Maintaining security guarantees of Confidential Computing within the WIMSE reference architecture calls for adding the extensions specified in this draft.
+Specifically, strong binding between the credential and the hosting enviroment, utilizing proof-of-possession of a workload credential backed by a key that is safeguarded against disclosure via data-in-use protections offered by Confidential Computing is a must-have.
+Additionally, invesion of the trust relationship between the workload and the hosting environment (where the workload cannot trust the hosting environment to attest it for purposes of obtaining a credential and must instead perform its own attestation) is another critical extension dictated by Confidential Computing.
```

Comment:
-Section in Line 222 is not clear at all, did you mean, in addition to underlying platform, the Workload MUST be capable to attest itself, to an independent third party Attestation Verification to establish the security guarantees required for establishing Trust demanded by TWI Architecture?

Reply:
What I mean to saw is that WIMSE expects the hosting environment to attest the workload, and under CC the workload must attest itself using underlying hardware platform's capabilities that Confidential Computing offers.

Reply:
ok, I will re-word the statement then, I will suggest a few edits to this PR!